workflow for developing zkSNARK dApp by zokrates on bitcoin/README.md

```diff
@@ -0,0 +1,71 @@
+# 比特币上基于 zokrates 的 zkSNARK 应用开发流程简介
+
+
+我们展示了[如何使用 zk-SNARKs 来开发比特币的信息不完整的游戏](https://xiaohuiliu.medium.com/incomplete-information-games-on-bitcoin-d79408050882)。
+
+
+众所周知，[ZoKrates](https://github.com/Zokrates/ZoKrates) 是 zkSNARKs 的工具箱。它可以帮助您在 DApp 中使用可验证的计算，从规范的基于高级语言的程序电路到生成计算证明，再到验证这些证明。
+
+
+在 [ZoKrates Fork 库](https://github.com/sCrypt-Inc/zokrates) 中，我们扩展了 ZoKrates 的功能。使其支持自动生成验证 zk-SNARKs 证明的sCrypt智能合约。
+
+
+## 安装
+
+下载并使用我们发布的二进制包:
+
+https://github.com/sCrypt-Inc/zokrates/releases/tag/v0.8.1
+
```

Comment:
You can also build from source.
```
git clone https://github.com/sCrypt/ZoKrates
cd ZoKrates
cargo +nightly build -p zokrates_cli --release
cd target/release
```


as in https://zokrates.github.io/gettingstarted.html#from-source